package.json

```diff
@@ -25,6 +25,7 @@
     "@angular/router": "~10.1.5",
     "@swimlane/ngx-datatable": "^18.0.0",
     "@webcomponents/custom-elements": "^1.4.2",
+    "ckeditor4-angular": "^2.0.0",
```

Comment:
I see there is also a ckeditor 5: https://www.npmjs.com/package/@ckeditor/ckeditor5-angular

Do we need the version 4 specifically or could we upgrade to 5?

Reply:
We want to stay with 4 as it provides built in features for source code editing which is not provided with version 5.  I experimented with both versoins.

https://ckeditor.com/docs/ckeditor4/latest/examples/sourcearea.html